packages/api/helper/nominationGsheetToDB.js

```diff
@@ -9,6 +9,7 @@ module.exports = function gsheetToDB() {
     // process.env.GOOGLE_SHEETS_PRIVATE_KEY,
     JSON.parse(process.env.GOOGLE_SHEETS_PRIVATE_KEY),
     ['https://www.googleapis.com/auth/spreadsheets']
+    // JSON.parse(process.env.GOOGLE_SHEETS_PRIVATE_KEY),
```

Comment:
Guessing we can revert this too

Reply:
To successfully retrieve google sheet data on my local machine, i have had to remove the JSON.parse leaving just the process.env.GOOGLE_SHEETS_PRIVATE_KEY, I am not certain of this.